python-sdk/nuscenes/eval/detection/render.py

```diff
@@ -87,6 +87,9 @@ def visualize_sample(nusc: NuScenes,
         if box.score >= conf_th:
             box.render(ax, view=np.eye(4), colors=('b', 'b', 'b'), linewidth=1)
 
+    # Add legend
+    ax.legend(['GT Box', 'EST Box'], loc='upper right')
```

Comment:
Let's make this an option, so that users can choose whether or not to render with the legend